<?xml version="1.0"?>
<launch>

    <rosparam file="$(find cob_controller_configuration_gazebo)/ros/controller/sdh_controller_cob.yaml" command="load" />

    <node name="sdh_controller_spawner" pkg="controller_manager" type="spawner" args="sdh_controller" />

    <!-- sdh simulated tactile sensors -->
    <include file="$(find schunk_simulated_tactile_sensors)/ros/launch/simulated_tactile_sensors.launch" />

</launch>
